src/ray/raylet/node_manager.cc

```diff
@@ -2188,9 +2209,18 @@ std::string NodeManager::DebugString() const {
     }
   }
   result << "\n- num live actors: " << live_actors;
-  result << "\n- num reconstructing actors: " << live_actors;
+  result << "\n- num reconstructing actors: " << reconstructing_actors;
   result << "\n- num dead actors: " << dead_actors;
   result << "\n- max num handles: " << max_num_handles;
+  stats::ActorStats().Record(live_actors,
+      {{stats::ValueTypeKey, "live_actors"}});
+  stats::ActorStats().Record(reconstructing_actors,
+      {{stats::ValueTypeKey, "reconstructing_actors"}});
+  stats::ActorStats().Record(dead_actors,
+      {{stats::ValueTypeKey, "dead_actors"}});
+  stats::ActorStats().Record(max_num_handles,
+      {{stats::ValueTypeKey, "max_num_handles"}});
```

Comment:
why record the metrics in `DebugString()` function? Better use a separate function to do this.